wfl/fit/gap_simple.py

```diff
@@ -4,7 +4,7 @@
 from pathlib import Path
```

Comment:
how about making a `gap` submodule? Then 
* `gap_simple.py` -> `gap/gap.py` or just in `gap/__init__.py` 
* `gap_multistage.py` -> `gap/multistage.py`
* `glue_2b.py` -> `gap/spline_2b.py` or `gap/glue.py` or `gap/glue_2b.py`
* `gap_relocate.py` -> `gap/relocate.py`

Reply:
Sounds good, although I don't like only gap simple in `__init__`.

Reply:
you don't like putting things in `__init__.py` or would you like to have fewer/more things in there than just `gap_simple.py` (if so, what?)?

Reply:
I don't like putting only gap_simple there.  I'd say put `gap_simple` in `simple.py`, and could go either way on having an empty `__init__` or having references to both simple and multistage.

Reply:
I guess I'll do this.

Reply:
I think this is done. @gelzinyte can you resolve this conversation if you think it's OK?

Reply:
looks good!